src/components/contextpopup/ContextPopupDirective.js

```diff
@@ -114,7 +114,6 @@ goog.require('ga_permalink');
               coord4326['lat'] = coord4326[1];
               scope.coordmgrs = $window.proj4.mgrs.forward(coord4326).
                   replace(/(.{5})/g, '$1 ');
-              scope.altitude = '-';
```

Comment:
As you 've seen after (in the partials) if there is no altitude value, the altitude will not show up. But I can revert back if you think the '-' is really wanted.